### ISSUE TITLE
Enable adding both repo and recipe from search results

### ISSUE THREAD

**Issue by homebysix** [lindegroup/autopkgr#260]
Currently the search panel only allows adding repos. Would be great to be able to add the recipe you searched for.

**Comment by eahrold** [comment]
I debated that, but it proved fairly tricky to recipe identifier is not reported during `autopkg search`. In the end I suspect most recipes end up as overrides anyway. 

I'll keep thinking and try to come up with good way to approach this.

**Comment by homebysix** [comment]
Coming back to this one — it still bugs me that even though I just clicked on a recipe in the search results, AutoPkgr still needs me to specify that I want to add that recipe to my list.

Could we create the following process?

1. User searches for a recipe and clicks on it in the search results.
2. An [Add repo and selected recipe] button would:
    1. Add the repo (as it does now)
    2. Use the result of `info` to obtain the full identifier of the new recipe. The equivalent of: <br />`autopkg info Clarify.pkg | awk '/Identifier/{print $2}'`
3. Adds the recipe identifier to the recipe_list.txt and checks its box in AutoPkgr.

I admit, there is the possibility of ambiguity — Firefox.pkg could refer to multiple recipes, each with different identifiers. But considering we know which repo we just added, it seems we should be able to figure out which recipe is the "right" one.
